core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/EmailAction.java

```diff
@@ -200,7 +200,7 @@ public PasswordAuthentication getPasswordAuthentication() {
       ClassLoader oldClassLoader = Thread.currentThread().getContextClassLoader();
       Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
       try {
-        Transport transport = session.getTransport(config.protocol);
+        Transport transport = session.getTransport(config.protocol.toLowerCase());
```

Comment:
the `Session.getTransport(String)` is case sensitive?
Then, what if there's some protocol that has an uppercase letter in it?